### ISSUE TITLE
Question: Does the script in test_attack.py only support GPU model? 

### ISSUE THREAD

**Issue by synergit** [cmhcbb/attackbox#13]
Repeat the Question: Does the script in test_attack.py only support GPU model? 
Thank you in advance.

**Comment by cmhcbb** [comment]
Yes. However, it could be modified compatible with CPU only easily with deleting all lines with .cuda.
